docs/book/src/tasks/experimental-features/runtime-sdk/implement-lifecycle-hooks.md

```diff
@@ -67,7 +67,7 @@ message: "error message if status == Failure"
 retryAfterSeconds: 10
 ```
 
-For additional details, refer to the [Draft OpenAPI spec](https://editor.swagger.io/?url=https://raw.githubusercontent.com/kubernetes-sigs/cluster-api/main/docs/proposals/images/runtime-hooks/runtime-hooks-openapi.yaml).
+For additional details, you can see the full schema in <button onclick="openSwaggerUI()">Swagger UI</button>.
```

Comment:
Okay so it looks like this works :).

The issue I'm solving here is that we need the URL of the `runtime-sdk-openapi.yaml` of the **current** book. 
The button just calculates that URL and opens the Swagger UI with it.

Reply:
Demo can be seen under:
* https://deploy-preview-6838--kubernetes-sigs-cluster-api.netlify.app/tasks/experimental-features/runtime-sdk/implement-lifecycle-hooks.html
* Click on one of the "Swagger UI" buttons

Reply:
Very fancy :slightly_smiling_face: